frontend/src/components/CheckInForm.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
+import { useCookies } from "react-cookie";
+
+const CheckInForm = ({onClose}) => {
+  const [name, setName] = useState("");
+  const [cookies, setCookie] = useCookies(["userId"]);
+  const [show, setShow] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setShow(false);
+    const userId = cookies.userId;
+    const payload = { userId: userId, userName: name };
+
+    fetch("/api/list", {
```

Comment:
In the future we'll have to make this a variable so we can easily change it out/test it when we release